src/components/topCarousel.tsx

```
@@ -0,0 +1,95 @@
+        ))}
+      </div>
+      <div className="topCarouselIndicator">
+        {posts.map((_, index) => (
+          <button
+            className={index === selectedIndex ? "active" : ""}
+            key={index}
+            onClick={() => {
+              if (!emblaApi) return;
+              emblaApi.scrollTo(index);
+            }}
+          />
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)